auth/src/main/java/com/firebase/ui/auth/ui/phone/VerifyPhoneNumberFragment.java

```diff
@@ -134,6 +135,8 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         // It is assumed that the phone number that are being wired in via Credential Selector
         // are e164 since we store it.
         Bundle params = getArguments().getBundle(ExtraConstants.PARAMS);
+        List<String> whitelistedCountryCodes = null;
```

Comment:
I think these two variables are not used?

Reply:
Removed them in my last commit.